examples/CMakeLists.txt

```diff
@@ -1527,6 +1527,26 @@ add_impactx_test(htu-beamline.py
     OFF  # no plot script yet
 )
 
+# HTU Beamline Model with Off-Energy Transport ##############
+#
+# no space charge
```

Comment:
Unnecessary/overly specific:
```suggestion
```
(I know we carry this around a few examples, but should clean it)